examples/service-nginx.json

```diff
@@ -1,11 +1,19 @@
 {
-  "id": "nginx-service",
   "kind": "Service",
-  "apiVersion": "v1beta1",
-  "protocol": "TCP",
-  "port": 80,
-  "containerPort": 80,
-  "selector": {
-    "name": "foo"
+  "apiVersion": "v1beta3",
+  "metadata": {
+    "name": "nginx"
+  },
+  "spec": {
+    "selector": {
+      "name": "nginx"
+    },
+    "ports": [
+      {
+        "protocol": "TCP",
```

Comment:
define a service port name here, will allow this to be used with a replicated nginx pod